packages/contract-wrappers/src/utils/zeroex_transaction_decoder.ts

```diff
@@ -0,0 +1,88 @@
+import { getContractAddressesForNetworkOrThrow, NetworkId } from '@0x/contract-addresses';
+import * as ContractArtifacts from '@0x/contract-artifacts';
+import { SimpleContractArtifact } from '@0x/types';
+import { AbiDefinition, ContractAbi } from 'ethereum-types';
+import * as _ from 'lodash';
+
+import {
+    DeployedContractInfo,
+    DeployedContractInfoByName,
+    TransactionData,
+    TransactionDecoder,
+    TransactionProperties,
+} from '@0x/utils';
+
+export class ZeroExTransactionDecoder extends TransactionDecoder {
```

Comment:
I think this should also be exposed by default on the contractWrappers instance. 

I think the name is a little confusing as well.  It doesn't decode transactions but abi encoded data.
`ZeroExCalldataDecoder`? What are your thoughts on this?